### PR TITLE
Refactored the stats

### PR DESCRIPTION
**Pull request by sherman-the-tank**
1) Remove the dependency on stats::Stats
2) Now each individual processor can have `specific counters

### CODE REVIEW

**Review by critical27 — COMMENTED** [review]
Generally LGTM, wait https://github.com/vesoft-inc/nebula-common/pull/392

File: src/storage/index/LookupProcessor.h [hunk 1]
```diff
@@ -13,15 +13,17 @@
 namespace nebula {
 namespace storage {
 
+extern ProcessorCounters kLookupCounters;
```
Comment (critical27) [inline src/storage/index/LookupProcessor.h:16]: Why not define all counters in `GraphStorageServiceHandler`?

Comment (sherman-the-tank) [reply src/storage/index/LookupProcessor.h:16]: Each individual processor could have it own counters. So it makes more sense to let the processors own the counters

**Review by panda-sheep — APPROVED** [review]
Good job, LGTM

**Review by critical27 — APPROVED** [review]
Well done

**Pull request merged**